meta/main.yml

```diff
@@ -21,15 +21,14 @@ galaxy_info:
     - stretch
   - name: Ubuntu
     versions:
-    - trusty
     - xenial
     - bionic
   - name: Fedora
     versions:
     - 25
     - 27
     - 28
-  - name: EL
+  - name: centos
```

Comment:
I looked up and it seems `centos` is not used on ansible-galaxy (EL comprises Centos too), therefore please remove this change.

Reply:
@Dhiraj240 can you update the PR please?